src/clusterfuzz/_internal/swarming/__init__.py

```diff
@@ -74,15 +67,24 @@ def _get_new_task_spec(command: str, job_name: str, download_url: str):
   # The startup script size in bytes
   digest_size_bytes = instance_spec['digest_size_bytes']
   # The service account that the task runs as.
-  service_account = instance_spec['service_account']
+  service_account = instance_spec['service_account_email']
+  # If this task request slice is not scheduled after waiting this long,
+  # the task state will be set to EXPIRED.
+  expiration_secs = instance_spec['expiration_secs']
+  # Maximum number of seconds the task can run before its process is
+  # forcibly terminated and the task results in TIMED_OUT.
+  execution_timeout_secs = instance_spec['execution_timeout_secs']
+  if command == 'fuzz':
+    execution_timeout_secs = swarming_config.get('fuzzing_session_duration')
```

Comment:
If this is set to the exact duration of the fuzzing session used by CF, won't this mean the task will always time out? I assume CF will run for just a little longer than this duration.

Reply:
Maybe I should rename this to `fuzzing_task_duration` instead because that's what I meant by this config entry

Reply:
Renamed to `fuzz_task_duration`. It is not exactly clear to me how long fuzz tasks run on batch currently. 
@jonathanmetzman  could you please advise on this?